src/resources/views/crew/show.blade.php

```diff
@@ -61,6 +62,17 @@ class="px-6 py-3 bg-gray-50 text-left text-xs font-medium text-gray-500
                                         {{ $crew->name }}
                                     </td>
                                 </tr>
+                                <tr class="border-b">
+                                    <th scope="col"
+                                        class="px-6 py-3 bg-gray-50 text-left text-xs font-medium text-gray-500
+                                        uppercase tracking-wider">
+                                        {{ __('crew/show.duplicate_algorithm') }}
+                                    </th>
+                                    <td class="px-6 py-4 whitespace-nowrap text-sm text-gray-900 bg-white
+                                    divide-y divide-gray-200">
+                                        {{ ListMatchingAlgorithm::find($crew->duplicate_algorithm_id)->name ?? null }}
```

Comment:
Don't access the name this way. 
Use relationships instead.


$crew->duplicate_algorithm->name
⚠️ you must define the relationship in the model first (https://laravel.com/docs/11.x/eloquent-relationships#one-to-one)